### PR TITLE
added my website for order of beverages

### PR DESCRIPTION
**Pull request by may-21**
This is with reference to csoc dev week 2 assignment.I have created a responsive website with proper logo,collapsible navbar,a preloader. It has three sections which consists of further subsections such that total number of pages is six. 
Responsiveness is added in the delivery address form.
Care has been taken to make the site compatible for small mobile screens.
It consists of internal javascript for the preloader.
Bootstrap is used for making the navbar.
Thanks for providing a good opportunity to learn.

### CODE REVIEW

File: mywebsite/index.htm [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		<p>Visit us at:Pandit Madan Mohan Malviya Rd, Ravidas Gate, Saket Nagar Colony, Lanka, Varanasi, Uttar Pradesh 221005</p>
+		<p>contact us:98894 87388</p>
+	<script src="scripts/main.js"></script>
+	<script>
+		var prel=document.getElementById('preloader');
+		function myFunction()
+		{
+			prel.style.display='none';
+		}
+	</script>
```
Comment (YashJipkate) [inline mywebsite/index.htm]: Does this work? I cannot see any preloader on opening your page.

File: mywebsite/index.htm [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <!-- Toggler/collapsibe Button -->
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <!-- Navbar links -->
+  <div class="collapse navbar-collapse" id="collapsibleNavbar">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="order now.html">
```
Comment (YashJipkate) [inline mywebsite/index.htm]: Refrain from naming having spaces.

File: mywebsite/index.htm [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        <a class="nav-link" href="rateus.html">
+        	<button type="button">Rate us!</button></a>
+      </li>
+      
+    </ul>
+    <ul class="nav navbar-nav navbar-right">
+      
+    </ul>
+  </div> 
+</nav>
```
Comment (YashJipkate) [inline mywebsite/index.htm]: Also, the nav bar is a bit off. Don't you think? I think you have modified it too much. The bootstrap seems overrun by some CSS class which is misaligning it.

Comment (may-21) [reply mywebsite/index.htm]: ya it's a bit off. I will take care next time.

File: mywebsite/index.htm (hunk 2 above)
Comment (may-21) [reply mywebsite/index.htm]: I will keep that in mind.

File: mywebsite/index.htm (hunk 1 above)
Comment (may-21) [reply mywebsite/index.htm]: yes the preloader works but it goes off quickly. You can navigate to the other pages to see the preloader.

**Pull request merged**